### DISCUSSION TITLE
Add documentation for changing default palette

### DISCUSSION THREAD

**Discussion by rdrpenguin04** [slint-ui/slint#683] [Q&A] [answered]
Hello!

I've started using SixtyFPS recently for porting an application to Rust, and one thing that I've noticed is that there doesn't seem to be a way to change the palette of the widgets. If I'm using any of the builtin styles, I am limited to whatever color palette it has, which is problematic when I'm trying to port an application with a very distinctive color scheme.

Let me know if there's any way I can be more specific or if I could help implement this feature!

**Comment by ogoffart** [comment, marked as answer]
Hi, 
Nice to hear that you are using SixtyFPS

There is currently no way to change the palette yet. This is tracked in issue #116 but this is not yet implemented.

In the mean time, you will have to basically re-implement the relevant widgets on your project to have their own Palette.
The widgets such as the button and such are fairly easy to do.  Otherwise you can just copy the fluent style for example, and change its palette.

(I converted the issue to a discussion because the issue is already tracked in #116, and we keep the issue tracker for actual things to do)

**Reply by rdrpenguin04** [reply]
Okay, thank you! I'll watch that issue and see what I can do; thank you for the quick response!
